types/pino/index.d.ts

```diff
@@ -601,12 +601,9 @@ declare namespace P {
     type WriteFn = (o: object) => void;
 
     interface LogDescriptor {
-        pid: number;
-        hostname: string;
         level: number;
-        time: string;
+        time: number;
         msg: string;
```

Comment:
Is `msg` always present?

Reply:
Based on the example in [the `mergingObject` documentation](https://getpino.io/#/docs/api?id=mergingobject-object), it looks like the message parameter is optional, and therefore the `msg` property should be optional to express this.

Furthermore, the property [can be configured to anything](https://getpino.io/#/docs/api?id=messagekey-string), so it's not a guarantee that the message property will always be `msg`.